workers/unity/Assets/Fps/Scripts/Config/MapBuilder.cs

```diff
@@ -400,6 +394,26 @@ private void MakeLevelObjectStatic()
             }
         }
 
+        private GameObject GetTileObjectAtCoordinate(Vector2Int coordinate)
+        {
+            var hits = Physics.OverlapSphere(GetWorldLocationFromCoordinate(coordinate), .5f);
+
+            foreach (var hit in hits)
+            {
+                var volume = hit.gameObject.GetComponent<TileTypeVolume>();
+                if (volume == null)
+                {
+                    continue;
+                }
+
+                return volume.TypeCollection.GetRandomTile();
+            }
+
+            return mapBuilderSettings.DefaultTileType == null
+                ? null
+                : mapBuilderSettings.DefaultTileType.GetRandomTile();
```

Comment:
If the default is null, we return null, else we return a random tile? 

That doesn't make a lot of sense? Surely we either return the default tile type always or return a random tile?

Reply:
DefaultTileType is a collection of 1 or more tiles, so it's,

If the default collection is null, we return null, otherwise we return a random tile from the default collection.

Reply:
Ohh wait, could this be a case for null propagation? `return mapBuilderSettings.DefaultTileType?.GetRandomTile();`?

Reply:
Yes - also can we rename `DefaultTileType` to `TileCollection` or similar then? 

If there are no overrides, the name Default doesn't make a ton of sense

Reply:
If I understand correctly, then there are overrides. If a space in the world is occupied by a TileTypeVolume, then that volume overrides the default tile type value.

Reply:
Ah yes you are correct, I wasn't looking at the broader context.

Can we rename to make it clear its a collection of tiles? `DefaultTileTypes`

Reply:
👍 I'll name it `DefaultTileTypeCollection`